### PR TITLE
Add a guide to PED-ANOVA for `n_trials>10000`

### PR DESCRIPTION
**Pull request by nabenabe0928**
## Motivation

As the information at:
- https://optuna.readthedocs.io/en/stable/reference/importance.html
- https://optuna.readthedocs.io/en/stable/reference/generated/optuna.importance.get_param_importances.html
hits when we google "optuna importance", I added the note about PED-ANOVA.

## Description of the changes

I added a note about PED-ANOVA.

### CODE REVIEW

File: docs/source/reference/importance.rst [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
 
 .. note::
 
    :class:`~optuna.importance.FanovaImportanceEvaluator` takes over 1 minute when given a study that contains 1000+ trials.
    We published `optuna-fast-fanova <https://github.com/optuna/optuna-fast-fanova>`_ library,
    that is a Cython accelerated fANOVA implementation. By using it, you can get hyperparameter
    importances within a few seconds.
+   If `n_trials` is more than 10000, the Cython implementation takes more than a minute, so you can use :class:`~optuna.importance.PedAnovaImportanceEvaluator` instead, enabling the evaluation to finish in a second.
```
Comment (nzw0301) [inline docs/source/reference/importance.rst:14]:
```suggestion
   If ``n_trials`` is more than 10000, the Cython implementation takes more than a minute, so you can use :class:`~optuna.importance.PedAnovaImportanceEvaluator` instead, enabling the evaluation to finish in a second.
```

File: optuna/importance/__init__.py [hunk 2]
```diff
@@ -72,6 +72,9 @@ def get_param_importances(
                 `optuna-fast-fanova <https://github.com/optuna/optuna-fast-fanova>`_ library,
                 that is a Cython accelerated fANOVA implementation.
                 By using it, you can get hyperparameter importances within a few seconds.
+                If `n_trials` is more than 10000, the Cython implementation takes more than
```
Comment (nzw0301) [inline optuna/importance/__init__.py:75]:
```suggestion
                If ``n_trials`` is more than 10000, the Cython implementation takes more than
```

**Review by nzw0301 — CHANGES_REQUESTED** [review]
Let me leave very minor comments.

**Comment by nabenabe0928** [comment]
@nzw0301 Thank you for your comments!

**Comment by nabenabe0928** [comment]
@HideakiImamura @eukaryo Could you review this PR?

**Review by eukaryo — APPROVED** [review]
LGTM!
fyi: Two modified pages build by CI are following:
https://optuna--5310.org.readthedocs.build/en/5310/reference/importance.html

https://optuna--5310.org.readthedocs.build/en/5310/reference/generated/optuna.importance.get_param_importances.html#optuna.importance.get_param_importances

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Pull request merged**